assets/add_new_asset.md

```diff
@@ -14,77 +49,88 @@ Token repository [https://github.com/trustwallet/assets](https://github.com/trus
+The `validators` folder contains folders: `assets` same structure as above and `list.json` information about validators. 
 
-`info` folder contains for now only `logo.png` that represents coin image
+The `blacklist.json` and `whitelist.json` files you may find in folders like `tron`, `ethereum` but not limited to, contain list of approved tokens and banned tokens.  Trust Wallet will never list blacklisted tokens, and only whitelisted tokens are shown in the token search results.
 
-`validators` folder contains folders: `assets` same structure as above and `list.json` information about validators. 
+#### Checksum format
+For Ethereum like networks, contract folders must be named according to the so-called **Checksum Format**, with mixed lowercase and uppercase letters, such as `0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359`.
+Non-checksum addresses (e.g. all lowercase) are considered invalid.
 
-`blacklist.json` and `whitelist.json` files you may find in folders like `tron`, `ethereum` but not limited to, contain list of address approved based on many criterias (TODO add criterias) and disapproved based on factors such as scam, outdated, abandoned contracts etc ... .
+You can find the checksum address by searching on [etherscan.io](https://etherscan.io),
+for example stablecoin [DAI](https://etherscan.io/address/0x89d24a6b4ccb1b6faa2625fe562bdd9a23260359)
+the checksum address is located at the top left corner of the page and has both uppercase and lowercase characters.
+Or [convert Ethereum address to Checksum address](https://piyolab.github.io/sushiether/RunScrapboxCode/?web3=1.0.0-beta.33&code=https://scrapbox.io/api/code/sushiether/web3.js_-_Ethereum_%E3%81%AE%E3%82%A2%E3%83%89%E3%83%AC%E3%82%B9%E3%82%92%E3%83%81%E3%82%A7%E3%83%83%E3%82%AF%E3%82%B5%E3%83%A0%E4%BB%98%E3%81%8D%E3%82%A2%E3%83%89%E3%83%AC%E3%82%B9%E3%81%AB%E5%A4%89%E6%8F%9B%E3%81%99%E3%82%8B/demo.js). 
 
+#### Layout
 
 ```
 .
 ├── blockchains
 │   └──ethereum
 │   │   └──assets
 │   │   │  └──0x0a2D9370cF74Da3FD3dF5d764e394Ca8205C50B6 // address folder
-│   │   │     └──logo.png // address logo
+│   │   │     └──logo.png  // token logo
+|   |   |     └──info.json // optional token info
 │   │   └──info
-│   │      └──logo.png // coin logo
+│   │   │  └──logo.png  // chain coin logo
+|   |   │  └──info.json // chain coin info
+│   │   └──whitelist.json // allow list for tokens
```

Comment:
why we did not rename it yet to `allowlist` and `denylist`? this is one of the first things I have suggested to do?

Reply:
I wanted to finish first changes with functional improvements, and this rename has to be also coordinated with backend.  It's done now, renamed here too.